app/views/scheduled_pickups/show.html.erb

```diff
@@ -5,7 +5,7 @@
   </header>
   <section class="zipcode-content-section">
     <h2 class="section-label">Scheduled Pickup Time</h2>
-    <span><%= format_pickup_time(@scheduled_pickup) %></span>
+    <span><%= @scheduled_pickup.time_range %></span>
```

Comment:
Yup, both [a feature spec](https://github.com/thoughtbot/freshfoodconnect/blob/7f8a3f97e91b72ac830c787cfdffb6887d2ba8f0/spec/features/admin_views_pickup_times_spec.rb#L16) and a [unit spec](https://github.com/thoughtbot/freshfoodconnect/blob/7f8a3f97e91b72ac830c787cfdffb6887d2ba8f0/spec/models/time_range_spec.rb)